src/test/codegen/used_with_arg.rs

```diff
@@ -0,0 +1,12 @@
+// compile-flags: -O
+
+#![crate_type = "lib"]
+#![feature(used_with_arg)]
+
+// CHECK: @llvm.used = appending global [1 x i8*]
```

Comment:
```suggestion
// CHECK: @llvm.used = appending global [1 x i8*]{{.*}}USED_LINKER
```
to check that the symbol does appear in there?